service/api_logic/filter_manager/base_filter_manager.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy.orm import Query
+
+class BaseFilterManager:
+    FILTERS ={}
+
+    @classmethod
+    def apply_filters(cls, query: Query, filters) -> Query:
+        filters_dict = filters._asdict()
```

Comment:
You are using a protected method here, which is not recommended in Python, I believe.

+ I hate dicts. Use models to describe your data. I bet you expect your data not to be totally random, right? If it's not random - don't treat it like it is. Describe the fields you wanna see and everything